client/dev/js/components/Home.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+require('../../../../scss/main.scss');
```

Comment:
When paths start getting long, webpack is your friend. you can resolve paths so its easier to reference.